unlock-app/src/services/ethpass.ts

```diff
@@ -0,0 +1,112 @@
+import { networks } from '@unlock-protocol/networks'
+import { storage } from '~/config/storage'
+
+export enum Platform {
+  APPLE = 'apple',
+  GOOGLE = 'google',
+}
+
+export const isEthPassSupported = (network: number) => {
+  // Check `chain` param on https://docs.ethpass.xyz/api-reference#tag/passes
+  return [1, 5, 137, 80001, 10, 69, 42161, 421611].indexOf(network) > -1
+}
+
+export const createWalletPass = async ({
+  signature,
+  lockAddress,
+  tokenId,
+  // image, ethPass does not support SVG for now...
+  signatureMessage,
+  network,
+  platform,
+}: any) => {
+  let pass = {}
+  if (platform === Platform.APPLE) {
+    pass = {
+      description: 'ETHPass Sample Pass',
+      auxiliaryFields: [],
+      backFields: [],
+      headerFields: [
+        {
+          key: 'header',
+          value: 'DEMO PASS',
+        },
+      ],
+      primaryFields: [],
+      secondaryFields: [
+        {
+          key: 'secondary1',
+          label: 'Lock Address',
+          value: `${lockAddress.slice(0, 6)}...${lockAddress.slice(
+            lockAddress.length - 4
+          )}`,
```

Comment:
Is this for minifying? Use minifyAddress function from ui library for consistency.